dev-tools/scripts/smokeTestRelease.py

```diff
@@ -277,16 +277,19 @@ def checkSigs(urlString, version, tmpDir, isSigned, keysFile):
   if os.path.exists(gpgHomeDir):
     shutil.rmtree(gpgHomeDir)
   os.makedirs(gpgHomeDir, 0o700)
-  run('gpg --homedir %s --import %s' % (gpgHomeDir, keysFile),
-      '%s/solr.gpg.import.log' % tmpDir)
+  gpgLogFile = '%s/solr.gpg.import.log' % tmpDir
+  run('gpg --homedir %s --import %s' % (gpgHomeDir, keysFile), gpgLogFile)
```

Comment:
This GPG stuff would be nicer to move after testChanges? or maybe in `isSigned`? If we aren't checking `isSigned` do we care about this gpg import?